pkg/query/api/v1.go

```diff
@@ -413,7 +434,7 @@ func (api *API) series(r *http.Request) (interface{}, *apiError) {
 	for _, mset := range matcherSets {
 		s, err := q.Select(mset...)
```

Comment:
no errReporter and warnings here?

Reply:
I actually almost deleted the endpoint – does anyone ever use it and for what? :)

Reply:
TBH I have no idea what this is for XD

Reply:
maybe let's kill it?

Reply:
I mean in theory it can give you a list of series without consulting any chunks for actual data... :) but not sure whether anyone relies on it for something